sql/core/src/test/resources/sql-tests/inputs/describe-table-column.sql

```diff
@@ -13,6 +13,8 @@ DESC FORMATTED desc_col_temp_table desc_col_temp_table.key;
 -- Describe a non-existent column
 DESC desc_col_temp_table key1;
 
+DROP VIEW desc_col_temp_table;
```

Comment:
since it's a view, can we name it `desc_col_temp_view`?

Reply:
and it's better to put all `DROP` commands at the end.

Reply:
ok. I'll fix it right now.